src/Razor/src/Microsoft.VisualStudio.Editor.Razor/Settings/ClientSettingsManager.cs

```diff
@@ -6,33 +6,36 @@
 using System.ComponentModel.Composition;
 using Microsoft.CodeAnalysis.ExternalAccess.Razor;
 using Microsoft.CodeAnalysis.Razor.Settings;
+using Microsoft.Extensions.Logging;
 using Microsoft.VisualStudio.Threading;
 
 namespace Microsoft.VisualStudio.Editor.Razor.Settings;
 
 [Export(typeof(IClientSettingsManager))]
-internal class ClientSettingsManager : IClientSettingsManager
+internal class ClientSettingsManager : IClientSettingsManager, IDisposable
 {
     public event EventHandler<ClientSettingsChangedEventArgs>? ClientSettingsChanged;
+    public event EventHandler<bool>? FeedbackRecordingChanged;
```

Comment:
If the in memory buffer ends up just logging all of the time, this can probably go. There isn't much conceptual difference between this event, and a `ClientSettingsChanged` even when `LogLevel` changes